src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/.template.config/template.json

```diff
@@ -23,23 +23,23 @@
   "primaryOutputs": [
     {
       "condition": "(UseWebAssembly && (HostIdentifier == \"dotnetcli\" || HostIdentifier == \"dotnetcli-preview\"))",
-      "path": "BlazorWeb-CSharp.sln"
+      "path": "BlazorWebCSharp.1.sln"
     },
     {
       "condition": "(UseWebAssembly && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "path": "BlazorWeb-CSharp/BlazorWeb-CSharp.csproj"
+      "path": "BlazorWebCSharp.1/BlazorWebCSharp.1.csproj"
     },
     {
       "condition": "(!UseWebAssembly)",
-      "path": "BlazorWeb-CSharp.csproj"
+      "path": "BlazorWebCSharp.1.csproj"
     },
     {
       "condition": "(UseWebAssembly && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "path": "BlazorWeb-CSharp.Client/BlazorWeb-CSharp.Client.csproj"
+      "path": "BlazorWebCSharp.1.Client/BlazorWebCSharp.1.Client.csproj"
     }
   ],
   "shortName": "blazor",
-  "sourceName": "BlazorWeb-CSharp",
+  "sourceName": "BlazorWebCSharp.1",
```

Comment:
This change was required in order to support all the default value forms for the `sourceName` in accordance with [the docs for `sourceName`](https://github.com/dotnet/templating/wiki/Naming-and-default-value-forms#sourcename-default-value-forms). Specifically, we want to use the `lower case class name` form for the value of `"applicationUrl"` in the *launchSettings.json* file.